ic-stable-structures/src/memory_utils.rs

```diff
@@ -5,7 +5,7 @@ use dfinity_stable_structures::{
     Memory,
 };
 
-#[cfg(all(not(target_arch = "wasm32")))]
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
we've started to use target_family wasm on evm, maybe we can use it also here

Reply:
done